molo/commenting/tests/test_admin.py

```diff
@@ -296,9 +296,11 @@ def test_export_csv(self):
         # not sure why user_name, user_email & article_full_url are blank
         # under test - they're in the export when testing manually
         expected_output = (
-            'submit_date,user_name,user_email,comment,parent,article_title,'
-            'article_subtitle,article_full_url,is_public,is_removed\r\n'
-            'datetime,,,export comment,,article 1,article 1 subtitle,,1,0\r\n'
+            'submit_date,user_name,user_email,comment,id,parent_id,'
+            'article_title,article_subtitle,article_full_url,'
+            'is_public,is_removed,parent\r\n'
```

Comment:
<img width="990" alt="screen shot 2017-03-31 at 4 42 21 pm" src="https://cloud.githubusercontent.com/assets/7100966/24555337/16c97c82-1631-11e7-8dc8-404ef8cef50f.png">
As a visual reference it's easier to have IDs next to each other, if it's next to its parent it loses some context